lib/filesystem/CArchiveLoader.cpp

```diff
@@ -24,10 +24,10 @@ ArchiveEntry::ArchiveEntry()
 
 }
 
-CArchiveLoader::CArchiveLoader(std::string _mountPoint, bfs::path _archive, bool extractArchives) :
+CArchiveLoader::CArchiveLoader(std::string _mountPoint, bfs::path _archive, bool _extractArchives) :
     archive(std::move(_archive)),
     mountPoint(std::move(_mountPoint)),
-	extractArchives(extractArchives)
+	extractArchives(_extractArchives)
```

Comment:
It is not a typo but valid c++ construct. First name is looked up as a class member and second as a parameter. we often use this feature. Or am I missing something?

Reply:
Looks like we do not have unified vision on what way is better)

Reply:
And our coding guidelines say nothing on this matter unfortunately.

Reply:
Huh, I though that initial report was about typo - self-initialization instead of using constructor parameter.

I'd say that change *is* an improvement, if only for consistency (method accepts 3 parameters, but only 2 start with underscore).

As for code style - I think that using same name for constructor parameters is perfectly fine. We should probably add this info into guidelines

Reply:
Now is consistent. Case closed :)